images/miq-app/Dockerfile

```diff
@@ -34,30 +32,17 @@ LABEL name="manageiq" \
 STOPSIGNAL SIGRTMIN+3
 
 ## Install EPEL repo, yum necessary packages for the build without docs, clean all caches
-RUN yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && \
-    yum -y install centos-release-scl-rh && \
+RUN yum -y install centos-release-scl-rh && \
     yum -y install --setopt=tsflags=nodocs \
-                   bison                   \
-                   bzip2                   \
-                   cmake                   \
                    file                    \
-                   gcc-c++                 \
-                   git                     \
```

Comment:
@Fryguy which of these need to be added back because they are also dependencies of ManageIQ?